op-plasma/cli.go

```diff
@@ -7,44 +7,55 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
-const (
-	EnabledFlagName         = "plasma.enabled"
-	DaServerAddressFlagName = "plasma.da-server"
-	VerifyOnReadFlagName    = "plasma.verify-on-read"
-	DaServiceFlag           = "plasma.da-service"
+var (
+	EnabledFlagName, EnabledFlagAlias                 = altDAFlags("enabled")
+	DaServerAddressFlagName, DaServerAddressFlagAlias = altDAFlags("da-server")
+	VerifyOnReadFlagName, VerifyOnReadFlagAlias       = altDAFlags("verify-on-read")
+	DaServiceFlag, DaServiceFlagAlias                 = altDAFlags("da-service")
 )
 
-func plasmaEnv(envprefix, v string) []string {
-	return []string{envprefix + "_PLASMA_" + v}
+// altDAFlags returns the flag names for altDA, with an Alias for plasma
+func altDAFlags(v string) (string, string) {
+	return fmt.Sprintf("altda.%s", v), fmt.Sprintf("plasma.%s", v)
```

Comment:
could also use simple concatenation
```suggestion
	return "altda." + v, "plasma." + v
```